examples/gen_tcp_server_example/rebar.config

```diff
@@ -0,0 +1,7 @@
+{erl_opts, [debug_info]}.
+{deps, [
+    {atomvm_lib, {git, "https://github.com/fadushin/atomvm_lib.git", {branch, "master"}}}
```

Comment:
URL needs update to github.com/atomvm/atomvm_lib

Reply:
Fixed and in the httpd example, which was also wrong (not in hex!)